scripts/freshvenvs.py

```diff
@@ -350,8 +349,8 @@ def generate_supported_versions(contrib_modules, all_used_versions):
             json_format = {
                 "dependency": dependency or contrib_module,
                 "integration": contrib_module,
-                "minimum_tracer_supported": str(ordered[-1]),
-                "max_tracer_supported": str(ordered[0]),
+                "minimum_tracer_tested": str(ordered[-1]),
```

Comment:
why "tracer_tested" ?

Reply:
just whatever i randomly chose, i can switch this